imagersite/imagersite/tests.py

```diff
@@ -1,21 +1,55 @@
 from django.test import TestCase, Client
+from django.core import mail
+from urllib.parse import urlparse
 
 
 class BasicViewTests(TestCase):
+    '''
+    Class for testing views
+    '''
     def test_home(self):
+        '''
+        Test home route
+        '''
         response = self.client.get('/')
         self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.templates[0].name, 'generic/home.html')
+        self.assertEqual(response.templates[1].name, 'generic/base.html')
```

Comment:
These asserts should be in a separate test from the status code assert statement because they are checking a different piece of the view functionality. Checking the status code validates that the route is hooked up properly. Checking the templates validates that your view function is working properly.